atomate/utils/utils.py

```diff
@@ -311,4 +312,37 @@ def recursive_update(d, u):
 def get_a_unique_id():
     ts = "{:.4f}".format(time())
     ts += str(randint(0,9999)).zfill(4)
-    return ts
\ No newline at end of file
+    return ts
+
+
+def get_uri(dir_name):
```

Comment:
* should probably go into utils/fileio.py
* make a static method of FileClient? (is this compatible with FileClient?)

Reply:
I don't think it is. FileClient needs the username and server address. While this gets a hostname, often we get the compute node address which is mostly useless. I think on further inspection, I would advocate eliminating it from the drone.